bin/get_parities.py

```diff
@@ -0,0 +1,43 @@
+import yaml
+from catlas.parity.parity_utils import get_parity_upfront
+from catlas.config_validation import config_validator
+import sys
+from jinja2 import Template
+import os
+import time
+
+# Load inputs and define global vars
+if __name__ == "__main__":
+
+    # Load the config yaml
+    config_path = sys.argv[1]
+    template = Template(open(config_path).read())
+    config = yaml.load(template.render(**os.environ), Loader=yaml.FullLoader)
+    if not config_validator.validate(config):
+        raise ValueError(
+            "Config has the following errors:\n%s"
+            % "\n".join(
+                [
+                    ": ".join(['"%s"' % str(i) for i in item])
+                    for item in config_validator.errors.items()
+                ]
+            )
+        )
+    else:
+        print("Config validated")
+    # Establish run information
+    run_id = time.strftime("%Y%m%d-%H%M%S") + "-" + config["output_options"]["run_name"]
+    os.makedirs(f"outputs/{run_id}/")
+
+    # Print catlas to terminal
+    with open("catlas/catlas_ascii.txt", "r") as f:
+        print(f.read())
+
+    # Generate parity plots
+    if ("make_parity_plots" in config["output_options"]) and (
+        config["output_options"]["make_parity_plots"]
+    ):
+        n_steps = get_parity_upfront(config, run_id)
```

Comment:
get_parity_upfront doesn't seem to return anything?

Reply:
it writes figures to the output folder

Reply:
but the value being assigned to n_steps is always going to be equal to None?